spec/compiler/codegen/pointer_spec.cr

```diff
@@ -474,4 +474,27 @@ describe "Code gen: pointer" do
       ptr == ptr
       )).to_b.should be_true
   end
+
+  it "takes pointerof lib external var" do
+    test_c(
+      %(
+        int external_var = 0;
+      ),
+      %(
+        lib LibFoo
+          $external_var : Int32
+        end
+
+        LibFoo.external_var = 1
+
+        ptr = pointerof(LibFoo.external_var)
+        x = ptr.value
+
+        ptr.value = 10
+
+        y = ptr.value
+
+        x + y
+      ), &.to_i.should eq(11))
```

Comment:
I'd prefer this spec to be more robust: we should check the value of `LibFoo.external_var` after setting it too. I think there should be a way to return multiple values from the test. Does returning a tuple work?

Reply:
Actually use a bool to coalesce the multiple tests.

Reply:
> we should check the value of LibFoo.external_var after setting it too

test does that:

```
ptr.value = 10

y = ptr.value
```

not understand

no way to test tuple. maybe you write test, i copy?

Reply:
It checks `ptr.value`, but not `LibFoo.external_var`.

Reply:
good point. updated